### PR TITLE
Fix for #94

### PR DESCRIPTION
**Pull request by ham1**
Resubmitting PR for #94 after doing a bad job of conflating two different issues on the same branch.

### CODE REVIEW

**Comment by ham1** [comment]
I've attempted a fix, any thoughts? The only problem I see is that it means the other fields are `null` rather than whatever they might have been initialised to, is there a way to `return field.get(pageObject)`?

File: htmlelements-java/src/test/java/ru/yandex/qatools/htmlelements/NonElementFieldsTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Graham Russell graham@ham1.co.uk
+ *         Date: 24.10.2015
+ */
+public class NonElementFieldsTest {
+
+    @Test
+    public void non_element_private_fields_should_not_be_decorated_as_elements() {
```
Comment (lanwen) [inline htmlelements-java/src/test/java/ru/yandex/qatools/htmlelements/NonElementFieldsTest.java]: please always use camelCase notation

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementDecorator.java [hunk 2]
```diff
@@ -50,26 +50,25 @@ public HtmlElementDecorator(CustomElementLocatorFactory factory) {
     }
 
     public Object decorate(ClassLoader loader, Field field) {
-        if (isTypifiedElement(field)) {
-            return decorateTypifiedElement(loader, field);
+        try {
```
Comment (lanwen) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/HtmlElementDecorator.java]: so, can you only add `try-catch` without changing the `if` style to `elif`? Previous variant is more readable for me

**Comment by ham1** [comment]
Updated.

**Pull request merged**